extensions/sql/lease-sql/src/main/java/org/eclipse/dataspaceconnector/sql/lease/SqlLease.java

```diff
@@ -36,7 +37,7 @@ public void setLeaseId(String leaseId) {
         this.leaseId = leaseId;
     }
 
-    public boolean isExpired() {
-        return getLeasedAt() + getLeaseDuration() < Instant.now().toEpochMilli();
+    public boolean isExpired(Clock clock) {
+        return getLeasedAt() + getLeaseDuration() < clock.millis();
```

Comment:
should this be `clock.toEpochMilli()`?

Reply:
That method doesn't exist. https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html#millis--

Reply:
based on coversation, clock.Instant.toMilli is the same as clock.millis